DataFormats/PortableTestObjects/src/alpaka/classes_serial.h

```diff
@@ -2,3 +2,15 @@
 #include "DataFormats/Portable/interface/Product.h"
 #include "DataFormats/PortableTestObjects/interface/TestHostCollection.h"
 #include "DataFormats/PortableTestObjects/interface/TestSoA.h"
+#include "DataFormats/Portable/interface/PortableCollectionCommon.h"
+
+namespace ROOT::Meta::Selection {
+  namespace PortableTestObjects {
+    class c0 : portablecollection::CollectionImpl<0, void> {};
+
+    using portablecollection::CollectionLeaf;
+    class CL0_TestHostMultiCollection : CollectionLeaf<0, portabletest::TestSoALayout<128, false>> {};
+    class CL1_TestHostMultiCollection : CollectionLeaf<1, portabletest::TestSoALayout2<128, false>> {};
+    class CL2_TestHostMultiCollection : CollectionLeaf<2, portabletest::TestSoALayout3<128, false>> {};
```

Comment:
It would be good to document the purpose of these class declarations.

Reply:
The corresponding XML definitions are now automatically commented in the generated XML for dictionary.